packages/scandipwa/src/component/GroupedProductsItem/GroupedProductsItem.container.js

```diff
@@ -59,8 +60,12 @@ export class GroupedProductsItemContainer extends PureComponent {
     }
 
     setQuantity(itemCount) {
-        const { setQuantity, product: { id } } = this.props;
-        setQuantity({ [id]: itemCount }, true);
+        const { setQuantity, product, product: { id } } = this.props;
+        const inStock = getProductInStock(product);
```

Comment:
Okey, but maybe better to not use 1 time variable, so you can just put it into if block conditional since it is already kind clearly see what it gets by name of util.